### PR TITLE
Create new package in apps directory for dapp console

### PR DESCRIPTION
**Pull request by imranjami**
### Description
- Creates a new next.js application in the `apps/dapp-console`
- Initialized with typescript + tailwind.css
- Wired up with the `ui-components` package

### CODE REVIEW

**Comment by imranjami** [comment]
Current dependencies on/for this PR:
* `main`
  * **PR #53** <a href="https://app.graphite.dev/github/pr/ethereum-optimism/ecosystem/53?utm_source=stack-comment-icon" target="_blank">Graphite</a>  👈

This [stack of pull requests](https://stacking.dev/?utm_source=stack-comment) is managed by [Graphite](https://app.graphite.dev/github/pr/ethereum-optimism/ecosystem/53?utm_source=stack-comment).

File: apps/dapp-console/.eslintrc.cjs [hunk 1]
```diff
@@ -0,0 +1,29 @@
+module.exports = {
```
Comment (imranjami) [inline apps/dapp-console/.eslintrc.cjs:1]: Using the same `.eslintrc` as the `bridge-app`

File: apps/dapp-console/package.json [hunk 2]
```diff
@@ -0,0 +1,30 @@
+{
+  "name": "dapp-console",
```
Comment (jakim929) [inline apps/dapp-console/package.json:2]: nit: rename to `@eth-optimism/dapp-console`

Comment (jakim929) [reply apps/dapp-console/package.json:2]: nit: can we add to `apps` section `nx.json` file?

File: apps/dapp-console/package.json [hunk 3]
```diff
@@ -0,0 +1,30 @@
...
+  "version": "0.1.0",
+  "private": true,
+  "scripts": {
+    "dev": "next dev",
+    "build": "next build",
+    "start": "next start",
+    "lint": "eslint \"**/*.{ts,tsx}\" && pnpm prettier --check \"**/*.{ts,tsx}\"",
+    "lint:fix": "eslint \"**/*.{ts,tsx}\" --fix --quiet && pnpm prettier \"**/*.{ts,tsx}\" --write --loglevel=warn"
```
Comment (jakim929) [inline apps/dapp-console/package.json:10]: nit: add typecheck commands

```
"typecheck": "tsc --noEmit"
```

**Review by jakim929 — APPROVED** [review]
Looks great! just a few nits

**Pull request merged**